x-pack/legacy/plugins/maps/server/maps_telemetry/telemetry_task.js

```diff
@@ -35,7 +35,7 @@ export function scheduleTask(server, taskManager) {
 }
 
 export function registerMapsTelemetryTask(server) {
-  const taskManager = server.taskManager;
+  const taskManager = server.plugins.task_manager;
```

Comment:
Same here, should check for null.

Reply:
Just checking in on the changes, which LGTM

A null check for this might be moot to do from the maps plugin, since the plugin definition `require`s `task_manager`: https://github.com/elastic/kibana/commit/55e7b183d75#diff-1bb2390b8f55e83f3a830c385a7d664eR21

It's odd that part ^^ worked before this PR